src/ensembl/src/shared/components/external-link/ExternalLink.test.tsx

```diff
@@ -31,21 +30,24 @@ const defaultProps: ExternalLinkProps = {
 };
 
 describe('<ExternalLink />', () => {
-  const renderExternalLink = (props: Partial<ExternalLinkProps> = {}) =>
-    render(<ExternalLink {...defaultProps} {...props} />);
-
+  const renderExternalLink = () => render(<ExternalLink {...defaultProps} />);
   it('renders without error', () => {
     const { container } = renderExternalLink();
     expect(() => container).not.toThrow();
   });
 
   it('applies the passed in classNames', () => {
     const { container } = renderExternalLink();
-    expect(container.getElementsByClassName('.icon')).toBeTruthy();
+
     expect(
       container
-        .querySelector('.link')
-        ?.classList.contains(defaultProps.classNames?.link as string)
+        .querySelector(`icon-mock`)
+        ?.getAttribute('classname')
+        ?.indexOf(defaultProps.classNames?.icon as string)
```

Comment:
If `indexOf` doesn't find anything, it returns `-1`, which is truthy. And if the string begins with a searched substring, the `indexOf` will be 0, which is falsy. Bad in both scenarios. I suppose what you were after was `.includes`?

Reply:
That's done